src/tailwind.preset.js

```diff
@@ -87,6 +87,8 @@ export default /** @type {Partial<import('tailwindcss').Config>} */ ({
       },
       spacing: {
         em: '1em',
+        '2em': '2em',
+        '4em': '4em',
```

Comment:
Does Tailwind not have a way to specify em-multiples like this without declaring all the multiples that you use?

Reply:
Tailwind's default sizing scale is `rem`-based. Most of our sizing uses that and doesn't require us to add values to the configuration. We need `em-`based sizing in just a few places, and I feel that being explicit about that in the preset values is actually helpful. It's likely this is the end of the line for em-unit sizing that we'll need.

There certainly are other options for doing this if it bugs yah.